src/main/resources/config/reflectionconfig-java.json

```diff
@@ -69,4 +87,48 @@
     "name":"java.nio.ByteOrder",
     "methods":[{"name":"nativeOrder","parameterTypes":[] }]
   }
+,
+  {
+    "name":"java.nio.file.attribute.PosixFilePermission",
+    "methods":[{"name":"values","parameterTypes":[] }]
+  }
+,
+  {
+    "name":"java.security.SecureRandomParameters"
+  }
+,
+  {
+    "name":"java.util.Random",
+    "fields":[{"name":"seed", "allowUnsafeAccess":true}]
+  }
+,
+  {
+    "name":"com.sun.crypto.provider.HmacSHA1",
+    "methods":[{"name":"<init>","parameterTypes":[] }]
+  }
+,
+  {
+    "name":"com.sun.security.sasl.gsskerb.JdkSASL",
```

Comment:
probably optional.

Reply:
What does optional mean? We don't have a way for the user to choose here.

Reply:
No, but an agent should be able to do. We can tackle this later.